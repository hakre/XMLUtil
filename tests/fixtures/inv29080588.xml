<soap:Envelope xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
               xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:soap="http://schemas.xmlsoap.org/soap/envelope/">
    <soap:Body>
        <Consult Localitation xmlns="Services/">
            <XMLin>
                &lt;ConsultXMLin Language=&quot;1&quot; xmlns:xsd=&quot;http://www.w3.org/2001/XMLSchema&quot; xmlns:xsi=&quot;http://www.w3.org/2001/XMLSchema-instance&quot;&gt;
                &lt;Consult&gt;
                &lt;Code&gt;XXXXX0700005020128012D&lt;/Code&gt;
                &lt;/Consult&gt;
                &lt;/ConsultXMLin&gt;</XMLin>
        </Consult Localitation></soap:Body>
</soap:Envelope>
