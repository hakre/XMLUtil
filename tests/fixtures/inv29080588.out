#001    <soap:Envelope xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"␊
#002                   xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:soap="http://schemas.xmlsoap.org/soap/envelope/">␊
#003        <soap:Body>␊
#004            <Consult Localitation xmlns="Services/">␊
[FATAL]                             ^- (41) Specification mandate value for attribute Localitation (4:29)
[FATAL]                             ^- (65) attributes construct error (4:29)
[FATAL]                             ^- (73) Couldn't find end of Start Tag Consult line 4 (4:29)
#005                <XMLin>␊
#006                    &lt;ConsultXMLin Language=&quot;1&quot; xmlns:xsd=&quot;http://www.w3.org/2001/XMLSchema&quot; xmlns:xsi=&quot;http://www.w3.org/2001/XMLSchema-instance&quot;&gt;␊
#007                    &lt;Consult&gt;␊
#008                    &lt;Code&gt;XXXXX0700005020128012D&lt;/Code&gt;␊
#009                    &lt;/Consult&gt;␊
#010                    &lt;/ConsultXMLin&gt;</XMLin>␊
#011            </Consult Localitation></soap:Body>␊
[FATAL]                  ^                        ^- (76) Opening and ending tag mismatch: Envelope line 1 and Body (11:43)
[FATAL]                  ^- (73) expected '>' (11:18)
[FATAL]                  ^- (76) Opening and ending tag mismatch: Body line 3 and Consult (11:18)
#012    </soap:Envelope>␊
[FATAL] ^- (5) Extra content at the end of the document (12:1)
#013    ␊
